made links to taigatribe.com work by prepending http://

Frazer opened the pull request:
On the KANBAN USER STORY pages, under where it says 'PUBLISH AS GIG IN TAIGA TRIBE'
![screen shot 2016-11-30 at 22 03 55](https://cloud.githubusercontent.com/assets/374464/20771232/e1057d7c-b748-11e6-97f2-64d37bd077bd.png)
there is a modal for  'More info'

In these modals is a link to taigatribe.com  that is a relative url, so it links to https://tree.taiga.io/project/my-project/us/taigatribe.com

![screen shot 2016-11-30 at 22 21 38](https://cloud.githubusercontent.com/assets/374464/20771858/87138824-b74b-11e6-8880-b4d53b87fa71.png)


In this pull request I have changed the links to http://taigatribe.com


Oh,  I just noticed that the links work when clicking 'open in a new tab', but do nothing when clicked normally  -  perhaps they need `target='newWindow'` . I am unsure how the links are being suppressed.  I created an issue for that. https://github.com/taigaio/taiga-front/issues/1178

Xaviju commented:
Hi @Frazer the issue is already fixed in master branch and will be in production as soon as possible. Thanks for reporting!

Frazer commented:
Do you mean   https://github.com/taigaio/taiga-front/commit/dbee1ef8c032012ede838a5032386a256a0afde1#diff-926d64f42ff55c8d5dafa47e65ac04b3

you only changed app/locales/taiga/locale-en.json

I think you'll find the problem in
	modified:   app/locales/taiga/locale-ca.json
	modified:   app/locales/taiga/locale-de.json
	modified:   app/locales/taiga/locale-en.json
	modified:   app/locales/taiga/locale-es.json
	modified:   app/locales/taiga/locale-fi.json
	modified:   app/locales/taiga/locale-fr.json
	modified:   app/locales/taiga/locale-it.json
	modified:   app/locales/taiga/locale-nb.json
	modified:   app/locales/taiga/locale-nl.json
	modified:   app/locales/taiga/locale-pl.json
	modified:   app/locales/taiga/locale-pt-br.json
	modified:   app/locales/taiga/locale-ru.json
	modified:   app/locales/taiga/locale-sv.json
	modified:   app/locales/taiga/locale-tr.json
	modified:   app/locales/taiga/locale-zh-hant.json

My pull request fixes all of them

Xaviju commented:
Hi @Frazer the translation files are generated automatically so should not be edited manually except for the locale-en version. We are also fixing the link color.
